portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Tim's STEP 2020 Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Tim's STEP 2020 Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>I am a software engineer at Google. I'm co-hosting three STEP interns this summer with
+          <a href="http://github.com/megankj">Megan</a>. Our pod members are
+          <ul>
+              <li><a href="http://github.com/lilymzhou">Lily</a>
+              <li><a href="http://github.com/ryansimone">Ryan</a>
+              <li><a href="http://github.com/shreyachatterjee00">Shreya</a>
+          </ul>
+      </p>
+      <p>
+          I have a cat and his name is Gunther.
```

Comment:
this is a test comment that maybe should be resolved before i approve?

Reply:
how do i make this blocking?

Reply:
I'm not sure if you can.

Reply:
how do i un-approve?